Apps/Contoso.WPF.Demo/App.xaml.cs

```diff
@@ -16,6 +17,13 @@ public partial class App
         protected override void OnStartup(StartupEventArgs e)
         {
             AppCenter.LogLevel = LogLevel.Verbose;
+            Crashes.GetErrorAttachments = (ErrorReport report) =>
+            {
+                return new ErrorAttachmentLog[]
+                {
+                    ErrorAttachmentLog.AttachmentWithText(Settings.Default.TextErrorAttachments, Settings.Default.FileErrorAttachments),
```

Comment:
@AnastasiaKubova Incorrect usage of API.

@guperrot It looks like it would be nice to add a note that second parameter `fileName` doesn't mean that the SDK will read content from file with this name.

Reply:
We haven't had any confusion reported by customers. 😓 
This is still a filename that is used by portal when it is downloaded.